### ISSUE TITLE
Unhandled UI Exception: NullReferenceException at JournalScan.DisplayStringFromRingClass

### ISSUE THREAD

**Issue by BryanBoru** [EDDiscovery/EDDiscovery#1253]
Got an error Unhandled UI Exemption was told to attach 
[Trace_20170906161835.0.log](https://github.com/EDDiscovery/EDDiscovery/files/1283046/Trace_20170906161835.0.txt):

```
[2017-09-06 17:56:36Z] 
==== UNHANDLED UI EXCEPTION ====
System.NullReferenceException: Object reference not set to an instance of an object.
   at EliteDangerousCore.JournalEvents.JournalScan.DisplayStringFromRingClass(String ringClass)
   at EliteDangerousCore.JournalEvents.JournalScan.RingInformation(Int32 ringno, Double scale, String scaletype)
   at EliteDangerousCore.JournalEvents.JournalScan.DisplayString(Int32 indent, Boolean includefront)
   at EDDiscovery.UserControls.UserControlScan.DrawNode(List`1 pc, ScanNode sn, Image notscanned, Point curpos, Size size, Int32& offset, Boolean aligndown, Int32 labelvoff)
   at EDDiscovery.UserControls.UserControlScan.DrawSystem()
   at EDDiscovery.UserControls.UserControlScan.NewEntry(HistoryEntry he, HistoryList hl)
   at System.Action`2.Invoke(T1 arg1, T2 arg2)
   at EDDiscovery.EDDiscoveryController.NewEntry(JournalEntry je)
   at EliteDangerousCore.EDJournalClass.ScanTickDone(List`1 entries)
==== cut ====
```

**Comment by BryanBoru** [comment]
happened again>

`[2017-09-06 20:15:12Z] Running logfile age check
[2017-09-06 20:15:12Z] Native library pre-loader is trying to load native SQLite library "C:\Program Files\EDDiscovery\EDDiscovery\x64\SQLite.Interop.dll"...
[2017-09-06 20:15:12Z] SQLite version 3.19.3
[2017-09-06 20:15:12Z] *** Elite Dangerous Discovery Initializing - Version 8.2.3.0 (Using EDDiscovery), Platform: Win32NT
[2017-09-06 20:15:12Z] Initializing database
[2017-09-06 20:15:12Z] Database initialization complete
[2017-09-06 20:15:13Z] Theme ID 10037
[2017-09-06 20:15:13Z] Unknown Gal Map object regional
[2017-09-06 20:15:13Z] Unknown Gal Map object organicPOI
[2017-09-06 20:15:13Z] Unknown Gal Map object geyserPOI
[2017-09-06 20:15:13Z] New watch on C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:13Z] Scanned C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:14Z] Unknown Gal Map object regional
[2017-09-06 20:15:14Z] Last message repeated 16 times
[2017-09-06 20:15:14Z] Unknown Gal Map object organicPOI
[2017-09-06 20:15:14Z] Last message repeated 7 times
[2017-09-06 20:15:14Z] Unknown Gal Map object geyserPOI
[2017-09-06 20:15:14Z] Last message repeated 43 times
[2017-09-06 20:15:16Z] Start Monitor on C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:18Z] Retrieving logs ending 07-Sep-17 12:15:16 AM
[2017-09-06 20:15:21Z] Retrieving logs ending 31-Aug-17 12:15:16 AM
[2017-09-06 20:15:23Z] EDSM Check date: 2017-09-06 22:19:13
[2017-09-06 20:15:24Z] Retrieving logs ending 24-Aug-17 12:15:16 AM
[2017-09-06 20:15:25Z] Import took 815ms
[2017-09-06 20:15:25Z] Id 2 count 24
[2017-09-06 20:15:25Z] Id 218 count 1
[2017-09-06 20:15:25Z] Id 300 count 4
[2017-09-06 20:15:25Z] Id 305 count 18
[2017-09-06 20:15:25Z] Id 306 count 6
[2017-09-06 20:15:25Z] Id 400 count 7
[2017-09-06 20:15:25Z] Id 409 count 17
[2017-09-06 20:15:25Z] Id 413 count 1
[2017-09-06 20:15:25Z] Id 508 count 18
[2017-09-06 20:15:25Z] Id 509 count 6
[2017-09-06 20:15:25Z] Id 512 count 7
[2017-09-06 20:15:25Z] Id 517 count 14
[2017-09-06 20:15:25Z] Id 608 count 55
[2017-09-06 20:15:25Z] Id 610 count 8
[2017-09-06 20:15:25Z] Id 611 count 1
[2017-09-06 20:15:25Z] Id 700 count 42
[2017-09-06 20:15:25Z] Id 707 count 2
[2017-09-06 20:15:25Z] Id 708 count 1
[2017-09-06 20:15:25Z] Id 709 count 5
[2017-09-06 20:15:25Z] Id 710 count 13
[2017-09-06 20:15:25Z] Id 711 count 2
[2017-09-06 20:15:25Z] Id 807 count 5
[2017-09-06 20:15:25Z] Id 808 count 17
[2017-09-06 20:15:25Z] Id 809 count 2
[2017-09-06 20:15:25Z] Id 810 count 106
[2017-09-06 20:15:25Z] Id 811 count 118
[2017-09-06 20:15:25Z] Id 812 count 50
[2017-09-06 20:15:25Z] Id 819 count 3
[2017-09-06 20:15:25Z] Id 907 count 9
[2017-09-06 20:15:25Z] Id 908 count 5
[2017-09-06 20:15:25Z] Id 909 count 5
[2017-09-06 20:15:25Z] Id 910 count 97
[2017-09-06 20:15:25Z] Id 911 count 31
[2017-09-06 20:15:25Z] Id 1009 count 1
[2017-09-06 20:15:25Z] Id 1010 count 28
[2017-09-06 20:15:25Z] Id 1011 count 18
[2017-09-06 20:15:25Z] Id 1012 count 6
[2017-09-06 20:15:25Z] Id 1013 count 37
[2017-09-06 20:15:25Z] Id 1105 count 7
[2017-09-06 20:15:25Z] Id 1112 count 28
[2017-09-06 20:15:25Z] Id 1208 count 8
[2017-09-06 20:15:25Z] Id 1209 count 40
[2017-09-06 20:15:25Z] Id 1210 count 2
[2017-09-06 20:15:25Z] Id 1211 count 1
[2017-09-06 20:15:25Z] Id 1212 count 38
[2017-09-06 20:15:25Z] Id 1213 count 21
[2017-09-06 20:15:25Z] Id 1214 count 6
[2017-09-06 20:15:25Z] Id 1217 count 4
[2017-09-06 20:15:25Z] Id 1218 count 16
[2017-09-06 20:15:25Z] Id 1305 count 58
[2017-09-06 20:15:25Z] Id 1308 count 6
[2017-09-06 20:15:25Z] Id 1310 count 1
[2017-09-06 20:15:25Z] Id 1311 count 2
[2017-09-06 20:15:25Z] Id 1315 count 5
[2017-09-06 20:15:25Z] Id 1318 count 19
[2017-09-06 20:15:25Z] Id 1405 count 27
[2017-09-06 20:15:25Z] Id 1406 count 16
[2017-09-06 20:15:25Z] Id 1412 count 22
[2017-09-06 20:15:25Z] Id 1416 count 22
[2017-09-06 20:15:25Z] Id 1504 count 37
[2017-09-06 20:15:25Z] Id 1505 count 37
[2017-09-06 20:15:25Z] Id 1511 count 2
[2017-09-06 20:15:25Z] Id 1512 count 41
[2017-09-06 20:15:25Z] Id 1515 count 1
[2017-09-06 20:15:25Z] Id 1516 count 4
[2017-09-06 20:15:25Z] Id 1603 count 4
[2017-09-06 20:15:25Z] Id 1604 count 36
[2017-09-06 20:15:25Z] Id 1605 count 15
[2017-09-06 20:15:25Z] Id 1606 count 12
[2017-09-06 20:15:25Z] Id 1609 count 7
[2017-09-06 20:15:25Z] Id 1610 count 33
[2017-09-06 20:15:25Z] Id 1613 count 4
[2017-09-06 20:15:25Z] Id 1614 count 12
[2017-09-06 20:15:25Z] Id 1702 count 32
[2017-09-06 20:15:25Z] Id 1703 count 14
[2017-09-06 20:15:25Z] Id 1704 count 12
[2017-09-06 20:15:25Z] Id 1801 count 6
[2017-09-06 20:15:25Z] Id 1802 count 1
[2017-09-06 20:15:25Z] Id 1804 count 46
[2017-09-06 20:15:25Z] Id 1812 count 45
[2017-09-06 20:15:25Z] Id 1906 count 23
[2017-09-06 20:15:25Z] Id 1907 count 15
[2017-09-06 20:15:25Z] Id 2101 count 14
[2017-09-06 20:15:25Z] Id 2300 count 34
[2017-09-06 20:15:25Z] Id 2504 count 4
[2017-09-06 20:15:25Z] Id 2505 count 12
[2017-09-06 20:15:25Z] Id 2516 count 7
[2017-09-06 20:15:25Z] Id 2517 count 13
[2017-09-06 20:15:25Z] Stop Monitor on C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:25Z] Scanned C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:26Z] Start Monitor on C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous
[2017-09-06 20:15:28Z] Retrieving logs ending 07-Sep-17 12:15:26 AM
[2017-09-06 20:15:31Z] Retrieving logs ending 31-Aug-17 12:15:26 AM
[2017-09-06 20:15:34Z] Retrieving logs ending 24-Aug-17 12:15:26 AM
[2017-09-06 20:15:37Z] Retrieving logs ending 17-Aug-17 12:15:26 AM
[2017-09-06 20:15:40Z] Retrieving logs ending 10-Aug-17 12:15:26 AM
[2017-09-06 20:15:42Z] Retrieving logs ending 03-Aug-17 12:15:26 AM
[2017-09-06 20:15:45Z] Retrieving logs ending 27-Jul-17 12:15:26 AM
[2017-09-06 20:15:47Z] Retrieving logs ending 20-Jul-17 12:15:26 AM
[2017-09-06 20:15:50Z] Retrieving logs ending 13-Jul-17 12:15:26 AM
[2017-09-06 20:15:52Z] Retrieving logs ending 06-Jul-17 12:15:26 AM
[2017-09-06 20:16:00Z] Retrieving logs ending 29-Jun-17 12:15:26 AM
[2017-09-06 20:16:03Z] Retrieving logs ending 22-Jun-17 12:15:26 AM
[2017-09-06 20:16:05Z] Retrieving logs ending 15-Jun-17 12:15:26 AM
[2017-09-06 20:16:08Z] Retrieving logs ending 08-Jun-17 12:15:26 AM
[2017-09-06 20:16:11Z] Retrieving logs ending 01-Jun-17 12:15:26 AM
[2017-09-06 20:16:13Z] Retrieving logs ending 25-May-17 12:15:26 AM
[2017-09-06 20:16:16Z] Retrieving logs ending 18-May-17 12:15:26 AM
[2017-09-06 20:16:18Z] Retrieving logs ending 11-May-17 12:15:26 AM
[2017-09-06 20:16:21Z] Retrieving logs ending 04-May-17 12:15:26 AM
[2017-09-06 20:16:24Z] Retrieving logs ending 27-Apr-17 12:15:26 AM
[2017-09-06 20:16:27Z] Retrieving logs ending 20-Apr-17 12:15:26 AM
[2017-09-06 20:16:29Z] Retrieving logs ending 13-Apr-17 12:15:26 AM
[2017-09-06 20:16:32Z] Retrieving logs ending 06-Apr-17 12:15:26 AM
[2017-09-06 20:16:34Z] Retrieving logs ending 30-Mar-17 12:15:26 AM
[2017-09-06 20:16:37Z] Retrieving logs ending 23-Mar-17 12:15:26 AM
[2017-09-06 20:16:40Z] Retrieving logs ending 16-Mar-17 12:15:26 AM
[2017-09-06 20:16:42Z] Retrieving logs ending 09-Mar-17 12:15:26 AM
[2017-09-06 20:16:45Z] Retrieving logs ending 02-Mar-17 12:15:26 AM
[2017-09-06 20:16:48Z] Retrieving logs ending 23-Feb-17 12:15:26 AM
[2017-09-06 20:16:50Z] Retrieving logs ending 16-Feb-17 12:15:26 AM
[2017-09-06 20:16:53Z] Retrieving logs ending 09-Feb-17 12:15:26 AM
[2017-09-06 20:16:56Z] Retrieving logs ending 02-Feb-17 12:15:26 AM
[2017-09-06 20:16:58Z] Retrieving logs ending 26-Jan-17 12:15:26 AM
[2017-09-06 20:17:01Z] Retrieving logs ending 19-Jan-17 12:15:26 AM
[2017-09-06 20:17:04Z] Retrieving logs ending 12-Jan-17 12:15:26 AM
[2017-09-06 20:17:06Z] Retrieving logs ending 05-Jan-17 12:15:26 AM
[2017-09-06 20:17:09Z] Retrieving logs ending 29-Dec-16 12:15:26 AM
[2017-09-06 20:17:11Z] Retrieving logs ending 22-Dec-16 12:15:26 AM
[2017-09-06 20:17:14Z] Retrieving logs ending 15-Dec-16 12:15:26 AM
[2017-09-06 20:17:17Z] Retrieving logs ending 08-Dec-16 12:15:26 AM
[2017-09-06 20:17:20Z] Retrieving logs ending 01-Dec-16 12:15:26 AM
[2017-09-06 20:17:22Z] Retrieving logs ending 24-Nov-16 12:15:26 AM
[2017-09-06 20:17:25Z] Retrieving logs ending 17-Nov-16 12:15:26 AM
[2017-09-06 20:17:28Z] Retrieving logs ending 10-Nov-16 12:15:26 AM
[2017-09-06 20:17:30Z] Retrieving logs ending 03-Nov-16 12:15:26 AM
[2017-09-06 20:17:33Z] Retrieving logs ending 27-Oct-16 12:15:26 AM
[2017-09-06 20:17:36Z] Retrieving logs ending 20-Oct-16 12:15:26 AM
[2017-09-06 20:17:37Z] Change in file, scan C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous\Journal.170906201736.01.log
[2017-09-06 20:17:39Z] Retrieving logs ending 13-Oct-16 12:15:26 AM
[2017-09-06 20:17:41Z] Retrieving logs ending 06-Oct-16 12:15:26 AM
[2017-09-06 20:17:44Z] Retrieving logs ending 29-Sep-16 12:15:26 AM
[2017-09-06 20:17:47Z] Retrieving logs ending 22-Sep-16 12:15:26 AM
[2017-09-06 20:17:50Z] Retrieving logs ending 15-Sep-16 12:15:26 AM
[2017-09-06 20:17:52Z] Retrieving logs ending 08-Sep-16 12:15:26 AM
[2017-09-06 20:17:55Z] Retrieving logs ending 01-Sep-16 12:15:26 AM
[2017-09-06 20:17:58Z] Retrieving logs ending 25-Aug-16 12:15:26 AM
[2017-09-06 20:18:00Z] Retrieving logs ending 18-Aug-16 12:15:26 AM
[2017-09-06 20:18:03Z] Retrieving logs ending 11-Aug-16 12:15:26 AM
[2017-09-06 20:18:06Z] Retrieving logs ending 04-Aug-16 12:15:26 AM
[2017-09-06 20:18:08Z] Retrieving logs ending 28-Jul-16 12:15:26 AM
[2017-09-06 20:18:11Z] Retrieving logs ending 21-Jul-16 12:15:26 AM
[2017-09-06 20:18:14Z] Retrieving logs ending 14-Jul-16 12:15:26 AM
[2017-09-06 20:18:16Z] Retrieving logs ending 07-Jul-16 12:15:26 AM
[2017-09-06 20:18:18Z] TLU C:\Users\Bryan\Saved Games\Frontier Developments\Elite Dangerous updated with commander 1
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM LoadGame
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:17:36 AM Fileheader
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM Cargo
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM Loadout
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM Materials
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM Rank
[2017-09-06 20:18:18Z] New entry 07-Sep-17 12:18:18 AM Progress
[2017-09-06 20:18:19Z] Retrieving logs ending 30-Jun-16 12:15:26 AM
[2017-09-06 20:18:22Z] Retrieving logs ending 23-Jun-16 12:15:26 AM
[2017-09-06 20:18:25Z] Retrieving logs ending 16-Jun-16 12:15:26 AM
[2017-09-06 20:18:25Z] New entry 07-Sep-17 12:18:25 AM Location
[2017-09-06 20:18:28Z] Retrieving logs ending 09-Jun-16 12:15:26 AM
[2017-09-06 20:18:31Z] Retrieving logs ending 02-Jun-16 12:15:26 AM
[2017-09-06 20:18:33Z] Retrieving logs ending 26-May-16 12:15:26 AM
[2017-09-06 20:18:36Z] Retrieving logs ending 19-May-16 12:15:26 AM
[2017-09-06 20:18:39Z] Retrieving logs ending 12-May-16 12:15:26 AM
[2017-09-06 20:18:42Z] Retrieving logs ending 05-May-16 12:15:26 AM
[2017-09-06 20:18:44Z] Retrieving logs ending 28-Apr-16 12:15:26 AM
[2017-09-06 20:18:47Z] Retrieving logs ending 21-Apr-16 12:15:26 AM
[2017-09-06 20:18:50Z] Retrieving logs ending 14-Apr-16 12:15:26 AM
[2017-09-06 20:18:52Z] Retrieving logs ending 07-Apr-16 12:15:26 AM
[2017-09-06 20:18:55Z] Retrieving logs ending 31-Mar-16 12:15:26 AM
[2017-09-06 20:18:58Z] Retrieving logs ending 24-Mar-16 12:15:26 AM
[2017-09-06 20:19:01Z] Retrieving logs ending 17-Mar-16 12:15:26 AM
[2017-09-06 20:19:03Z] Retrieving logs ending 10-Mar-16 12:15:26 AM
[2017-09-06 20:19:06Z] Retrieving logs ending 03-Mar-16 12:15:26 AM
[2017-09-06 20:19:09Z] Retrieving logs ending 25-Feb-16 12:15:26 AM
[2017-09-06 20:19:09Z] New entry 07-Sep-17 12:19:09 AM StartJump
[2017-09-06 20:19:11Z] Retrieving logs ending 18-Feb-16 12:15:26 AM
[2017-09-06 20:19:14Z] Retrieving logs ending 11-Feb-16 12:15:26 AM
[2017-09-06 20:19:14Z] New entry 07-Sep-17 12:19:14 AM SupercruiseEntry
[2017-09-06 20:19:17Z] Retrieving logs ending 04-Feb-16 12:15:26 AM
[2017-09-06 20:19:20Z] Retrieving logs ending 28-Jan-16 12:15:26 AM
[2017-09-06 20:19:22Z] Retrieving logs ending 21-Jan-16 12:15:26 AM
[2017-09-06 20:19:25Z] Retrieving logs ending 14-Jan-16 12:15:26 AM
[2017-09-06 20:19:28Z] Retrieving logs ending 07-Jan-16 12:15:26 AM
[2017-09-06 20:19:31Z] Retrieving logs ending 31-Dec-15 12:15:26 AM
[2017-09-06 20:19:33Z] Retrieving logs ending 24-Dec-15 12:15:26 AM
[2017-09-06 20:19:36Z] Retrieving logs ending 17-Dec-15 12:15:26 AM
[2017-09-06 20:19:39Z] Retrieving logs ending 10-Dec-15 12:15:26 AM
[2017-09-06 20:19:42Z] Retrieving logs ending 03-Dec-15 12:15:26 AM
[2017-09-06 20:19:44Z] Retrieving logs ending 26-Nov-15 12:15:26 AM
[2017-09-06 20:19:47Z] Retrieving logs ending 19-Nov-15 12:15:26 AM
[2017-09-06 20:19:50Z] Retrieving logs ending 12-Nov-15 12:15:26 AM
[2017-09-06 20:19:53Z] Retrieving logs ending 05-Nov-15 12:15:26 AM
[2017-09-06 20:19:55Z] Retrieving logs ending 29-Oct-15 12:15:26 AM
[2017-09-06 20:19:58Z] Retrieving logs ending 22-Oct-15 12:15:26 AM
[2017-09-06 20:20:01Z] Retrieving logs ending 15-Oct-15 12:15:26 AM
[2017-09-06 20:20:03Z] Retrieving logs ending 08-Oct-15 12:15:26 AM
[2017-09-06 20:20:06Z] Retrieving logs ending 01-Oct-15 12:15:26 AM
[2017-09-06 20:20:09Z] Retrieving logs ending 24-Sep-15 12:15:26 AM
[2017-09-06 20:20:11Z] Retrieving logs ending 17-Sep-15 12:15:26 AM
[2017-09-06 20:20:14Z] Retrieving logs ending 10-Sep-15 12:15:26 AM
[2017-09-06 20:20:17Z] Retrieving logs ending 03-Sep-15 12:15:26 AM
[2017-09-06 20:20:19Z] Retrieving logs ending 27-Aug-15 12:15:26 AM
[2017-09-06 20:20:22Z] Retrieving logs ending 20-Aug-15 12:15:26 AM
[2017-09-06 20:20:25Z] Retrieving logs ending 13-Aug-15 12:15:26 AM
[2017-09-06 20:20:27Z] Retrieving logs ending 06-Aug-15 12:15:26 AM
[2017-09-06 20:20:30Z] Retrieving logs ending 30-Jul-15 12:15:26 AM
[2017-09-06 20:20:33Z] Retrieving logs ending 23-Jul-15 12:15:26 AM
[2017-09-06 20:20:36Z] Retrieving logs ending 16-Jul-15 12:15:26 AM
[2017-09-06 20:20:38Z] Retrieving logs ending 09-Jul-15 12:15:26 AM
[2017-09-06 20:20:41Z] Retrieving logs ending 02-Jul-15 12:15:26 AM
[2017-09-06 20:20:44Z] Retrieving logs ending 25-Jun-15 12:15:26 AM
[2017-09-06 20:20:47Z] Retrieving logs ending 18-Jun-15 12:15:26 AM
[2017-09-06 20:20:50Z] Retrieving logs ending 11-Jun-15 12:15:26 AM
[2017-09-06 20:20:52Z] Retrieving logs ending 04-Jun-15 12:15:26 AM
[2017-09-06 20:20:55Z] Retrieving logs ending 28-May-15 12:15:26 AM
[2017-09-06 20:20:58Z] Retrieving logs ending 21-May-15 12:15:26 AM
[2017-09-06 20:21:00Z] Retrieving logs ending 14-May-15 12:15:26 AM
[2017-09-06 20:21:03Z] Retrieving logs ending 07-May-15 12:15:26 AM
[2017-09-06 20:21:06Z] Retrieving logs ending 30-Apr-15 12:15:26 AM
[2017-09-06 20:21:09Z] Retrieving logs ending 23-Apr-15 12:15:26 AM
[2017-09-06 20:21:11Z] Retrieving logs ending 16-Apr-15 12:15:26 AM
[2017-09-06 20:21:14Z] Retrieving logs ending 09-Apr-15 12:15:26 AM
[2017-09-06 20:21:17Z] Retrieving logs ending 02-Apr-15 12:15:26 AM
[2017-09-06 20:21:19Z] Retrieving logs ending 26-Mar-15 12:15:26 AM
[2017-09-06 20:21:22Z] Retrieving logs ending 19-Mar-15 12:15:26 AM
[2017-09-06 20:21:25Z] Retrieving logs ending 12-Mar-15 12:15:26 AM
[2017-09-06 20:21:27Z] Retrieving logs ending 05-Mar-15 12:15:26 AM
[2017-09-06 20:21:30Z] Retrieving logs ending 26-Feb-15 12:15:26 AM
[2017-09-06 20:21:33Z] Retrieving logs ending 19-Feb-15 12:15:26 AM
[2017-09-06 20:21:36Z] Retrieving logs ending 12-Feb-15 12:15:26 AM
[2017-09-06 20:21:38Z] Retrieving logs ending 05-Feb-15 12:15:26 AM
[2017-09-06 20:21:41Z] Retrieving logs ending 29-Jan-15 12:15:26 AM
[2017-09-06 20:21:44Z] Retrieving logs ending 22-Jan-15 12:15:26 AM
[2017-09-06 20:21:46Z] Retrieving logs ending 15-Jan-15 12:15:26 AM
[2017-09-06 20:21:49Z] Retrieving logs ending 08-Jan-15 12:15:26 AM
[2017-09-06 20:21:52Z] Retrieving logs ending 01-Jan-15 12:15:26 AM
[2017-09-06 20:21:55Z] Retrieving logs ending 25-Dec-14 12:15:26 AM
[2017-09-06 20:21:57Z] Retrieving logs ending 18-Dec-14 12:15:26 AM
[2017-09-06 20:22:00Z] Retrieving logs ending 11-Dec-14 12:15:26 AM
[2017-09-06 20:22:03Z] Retrieving logs ending 04-Dec-14 12:15:26 AM
[2017-09-06 20:22:05Z] Retrieving logs ending 27-Nov-14 12:15:26 AM
[2017-09-06 20:22:21Z] New entry 07-Sep-17 12:22:21 AM Scan
[2017-09-06 20:22:22Z] Update sync flag ID 9993 with 2
[2017-09-06 20:23:30Z] New entry 07-Sep-17 12:23:29 AM StartJump
[2017-09-06 20:23:48Z] New entry 07-Sep-17 12:23:48 AM FSDJump
[2017-09-06 20:23:48Z] Arrived at system: Hyades Sector DG-X c1-7 1:th visit.
[2017-09-06 20:23:49Z] Update sync flag ID 9995 with 2
[2017-09-06 20:23:49Z] Update sync flag ID 9995 with 3
[2017-09-06 20:24:05Z] New entry 07-Sep-17 12:24:05 AM FuelScoop
[2017-09-06 20:24:06Z] 
==== UNHANDLED UI EXCEPTION ====
System.NullReferenceException: Object reference not set to an instance of an object.
   at EliteDangerousCore.JournalEvents.JournalScan.DisplayStringFromRingClass(String ringClass)
   at EliteDangerousCore.JournalEvents.JournalScan.RingInformation(Int32 ringno, Double scale, String scaletype)
   at EliteDangerousCore.JournalEvents.JournalScan.DisplayString(Int32 indent, Boolean includefront)
   at EDDiscovery.UserControls.UserControlScan.DrawNode(List`1 pc, ScanNode sn, Image notscanned, Point curpos, Size size, Int32& offset, Boolean aligndown, Int32 labelvoff)
   at EDDiscovery.UserControls.UserControlScan.DrawSystem()
   at EDDiscovery.UserControls.UserControlScan.NewEntry(HistoryEntry he, HistoryList hl)
   at System.Action`2.Invoke(T1 arg1, T2 arg2)
   at EDDiscovery.EDDiscoveryController.NewEntry(JournalEntry je)
   at EliteDangerousCore.EDJournalClass.ScanTickDone(List`1 entries)
==== cut ====
[2017-09-06 20:26:03Z] New entry 07-Sep-17 12:26:03 AM Scan
[2017-09-06 20:26:03Z] Update sync flag ID 9997 with 2
[2017-09-06 20:30:28Z] Retrieving logs starting 07-Sep-17 12:15:26 AM
[2017-09-06 20:33:52Z] New entry 07-Sep-17 12:33:52 AM Scan
[2017-09-06 20:33:53Z] Update sync flag ID 9998 with 2`

**Comment by robbyxp1** [comment]
Fixed in master.

**Comment by phroggster** [comment]
Migrated commentary from #1258:

Klightspeed:
> robbyxp1 finwen should we create an 8.2.6 release with this bugfix in it?

[Me]:
> I'm in agreement that a release should be made that includes this fix, but I'm indifferent as to whether it's tagged 8.2.6 or 8.3.0.

**Comment by Obey138** [comment]
Can we get the next release ASAP or at least a working release with just this fix?

**Comment by robbyxp1** [comment]
Finwen will have to comment.  needs to be a 8.2 series.

**Comment by klightspeed** [comment]
This should be fixed in https://github.com/EDDiscovery/EDDiscovery/releases/tag/Release_8.2.6

**Comment by phroggster** [comment]
Closing this issue, as it has been resolved in v8.2.7/v8.3.1 via c1a81dc83df1950e19a39d936aebc5fa2e911925.
